cookbooks/muparser_temperature_example/muparser_temp_ex.prm

```diff
@@ -0,0 +1,168 @@
+# Basic parameters file for using a function expression to define the
+# initial temperature structure using (? : ) syntax for if-else conditions.
+# 2D, Dimensional, Boussinesq
+# Simple Material Model
+
+# Dimension, end-time (years) and output directory
+# Only does zeroth time-step to show that initial condition works
+set Dimension                              = 2
+set Use years in output instead of seconds = true
+set Start time                             = 0
+set End time                               = 0
+set Output directory                       = output-muparser_temp_example
+
+# Parameters Related to the Solver and Pressure Normalization
+# Normalization of the temperature
+set Pressure normalization                 = surface
+set Surface pressure                       = 0  
+
+# Default values are 1e-12 and 1e-7
+# Using larger values so this will just define initial condition and exit
+subsection Solver parameters
+  set Temperature solver tolerance         = 1e-6
+  
+  subsection Stokes solver parameters
+    set Linear solver tolerance            = 1e-3
+  end
+end
+
+
+# FORM OF THE EQUATIONS TO USE
+# - can set this as the formulation:
```

Comment:
This is not quite clear to me, maybe reformulate this to: `The following is equivalent to setting: ... . But we will choose to set this explicitly to make it easier to change this later.